runtime/doc/diff.txt

```diff
@@ -1,39 +1,31 @@
-*diff.txt*      For Vim version 7.4.  Last change: 2014 May 20
+*diff.txt*      For Vim version 7.4.  Last change: 2015 Jan 19
 
 
 		  VIM REFERENCE MANUAL    by Bram Moolenaar
 
 
-				*diff* *vimdiff* *gvimdiff* *diff-mode*
-This file describes the |+diff| feature: Showing differences between two,
-three or four versions of the same file.
+							*diff* *diff-mode*
+This file describes |diff-mode|, which shows the differences between two,
+three, or four versions of the same file.
 
 The basics are explained in section |08.7| of the user manual.
 
-1. Starting diff mode		|vimdiff|
+1. Starting diff mode		|diff-mode|
 2. Viewing diffs		|view-diffs|
 3. Jumping to diffs		|jumpto-diffs|
 4. Copying diffs		|copy-diffs|
 5. Diff options			|diff-options|
 
-{not in Vi}
-
 ==============================================================================
 1. Starting diff mode
 
-The easiest way to start editing in diff mode is with the "vimdiff" command.
-This starts Vim as usual, and additionally sets up for viewing the differences
-between the arguments. >
-
-	vimdiff file1 file2 [file3 [file4]]
-
-This is equivalent to: >
+To start editing in diff mode, start Nvim with "nvim -d".  This starts Nvim
+and sets up for viewing the differences between the arguments. >
 
-	vim -d file1 file2 [file3 [file4]]
+    $ nvim -d file1 file2 [file3 [file4]]
```

Comment:
`$` is not worth the space, and may confuse Windows users. Plus it will cause the rest of the documentation to be inconsistent.